api/v1alpha1/sandbox_types.go

```diff
@@ -35,6 +45,17 @@ type SandboxSpec struct {
 
 // SandboxStatus defines the observed state of Sandbox.
 type SandboxStatus struct {
+	// FirstReadyTime - when did the sandbox become ready first
+	FirstReadyTime *metav1.Time `json:"firstReadyTime,omitempty"`
```

Comment:
Just curious: What's the usage of this `"first"ReadyTime`? Is there another `readyTime`?

Reply:
Makes sense! Then the "TTL from" for that should probably be "firstReady" too?